src/main/java/com/kakao/cafe/controller/ArticleController.java

```diff
@@ -0,0 +1,37 @@
+package com.kakao.cafe.controller;
+
+import com.kakao.cafe.service.ArticleService;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+
+@Controller
+public class ArticleController {
+
+  private final ArticleService articleService;
+
+  public ArticleController(ArticleService articleService) {
+    this.articleService = articleService;
+  }
+
+  @GetMapping("/qna/new")
+  public String questionForm() {
+    return "qna/form";
+  }
+
+  @PostMapping("/qnaList")
```

Comment:
```suggestion
  @PostMapping("/qna")
```
POST에 생성한다라는 의미가 있기 때문에 요 정도로 하면 좋을 것 같아요.

Reply:
아아..넵!!